### PR TITLE
6 get article comments

### PR DESCRIPTION
**Pull request by ELR143**

### CODE REVIEW

**Review by popatre — CHANGES_REQUESTED** [review]
Some good stuff - missing some tests though:

- [x] Status 200: OK
Responds with empty array if article_id is valid but has no associated comments.
- [x] Status 400: Bad request, invalid article_id
E.g. GET /api/articles/notanumber/comments

- [x] Check youre getting the correct id

File: models/news-model.js [hunk 1]
```diff
@@ -27,9 +27,7 @@ exports.selectArticleById = (id) => {
 exports.selectCommentsByArticleId = (id) => {
   const query = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`;
   return db.query(query, [id]).then((articleComments) => {
-    if (articleComments.rows.length === 0) {
-      return Promise.reject();
-    }
+    console.log(articleComments.rows);
```
Comment (ghost) [inline models/news-model.js]: 👀

File: models/utils-model.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+const format = require("pg-format");
+
+exports.checkExists = (table, column, value) => {
+  const query = format(`SELECT * FROM %I WHERE %I = $1;`, table, column);
+  return db.query(query, [value]).then((commentsArray) => {
+    if (commentsArray.rows.length === 0) {
+      return Promise.reject();
+    }
```
Comment (ghost) [inline models/utils-model.js:9]: Niiiiice

**Review by ghost — APPROVED** [review]
Nicely done. Merge away!

**Pull request merged**